### PR TITLE
EOS-19470: Elasticsearch and Kibana deployment using Opendistro

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise <yashodhan.pise@seagate.com>

Opendistro internally has a direct dependency on elasticsearch-oss.
Thus, the original problem is unaddressed. Hence, putting this PR on hold.

### CODE REVIEW

**Comment by ypise** [comment]
Opendistro internally has a direct dependency on elasticsearch-oss.
Thus, the original problem is unaddressed. Hence, putting this PR on hold.

**Review by mazinamdar — APPROVED** [review]
looks good to me

**Review by 83bhp — APPROVED** [review]
Looks ok to me, make sure its tested before merging!

File: srv/components/misc_pkgs/kibana/config.sls [hunk 1]
```diff
@@ -26,6 +26,10 @@ Configure Kibana service:
     - name: /etc/systemd/system/kibana.service
     - source: salt://components/misc_pkgs/kibana/files/kibana.service
 
+Remove kibana security plugin:
+  cmd.run:
+    - name: "sudo /usr/share/kibana/bin/kibana-plugin remove opendistroSecurityKibana --allow-root"
```
Comment (ypise) [inline srv/components/misc_pkgs/kibana/config.sls:31]: @tanujashinde0405 
`sudo` is unnecessary here. JFYI

Comment (tanujashinde0405) [reply srv/components/misc_pkgs/kibana/config.sls:31]: okay

**Pull request merged**